dali/operators/image/distortion/jpeg_compression_distortion_op_cpu.cc

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <vector>
+#include <opencv2/opencv.hpp>
+#include "dali/operators/image/distortion/jpeg_compression_distortion_op.h"
+
+namespace dali {
+
+DALI_SCHEMA(JpegCompressionDistortion)
+    .DocStr(R"code(Produces JPEG-like distortion in RGB images.
+
+The level of degradation of the image can be controlled with the ``quality`` argument,
+)code")
+    .NumInput(1)
+    .InputLayout(0, "HWC")
+    .NumOutput(1)
+    .AddOptionalArg("quality",
+        R"code(JPEG compression quality from 1 (lowest quality) to 95 (highest quality).
+
+Any values outside the range 1-99 will be clamped.)code",
```

Comment:
```suggestion
Any values outside the range 1-95 will be clamped.)code",
```
?

Reply:
It was actually the other way around. I fixed it now

Reply:
I think that you can set quality `100` in many graphics editors. Also, I predict that having this parameter as `float` would be more convenient.

Reply:
Both libjpeg and OpenCV use integer for the quality factor. I will check if 100 makes sense in the current formula.